mkosi/distributions/debian.py

```diff
@@ -124,7 +112,6 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bo
         setup_apt(state, cls.repositories(state))
         invoke_apt(state, "update", apivfs=False)
         invoke_apt(state, "install", packages, apivfs=apivfs)
-        install_apt_sources(state, cls.repositories(state, local=False))
```

Comment:
So I still don't think we've made this redundant since there's no easy way for debian users to get the default mkosi settings into the image. We can still discuss this, but let's do that in a separate PR